### DISCUSSION TITLE
"Minimum Motion Frames" unit of measure: frames or seconds?

### DISCUSSION THREAD

**Discussion by PieBru** [Motion-Project/motion#1491] [General]
Hi,
MotionEye on Arch Linux and MotionEyeOS cams here.

As most people measure time in seconds instead of fps, IMO it would be better to measure "Minimum Motion" as seconds instead of frames, as proposed [here](https://github.com/ccrisan/motioneye/issues/1355).

@ccrisan (MotionEye* maintainer) observed the actual frame rate varies depending on the camera configuration. If it is set to automatic exposure mode, the actual frame rate will vary depending on lighting conditions. Poor lighting condition means longer exposure means lower frame rate.

Measuring it in seconds, as most commercial products already do, will lead to easier config maintenance over time.

Is it already possible?
Thanks,
Piero

**Comment by baudev** [comment]
Moreover, in the current version of the documentation
https://motion-project.github.io/motion_config.html#minimum_motion_frames, the unit is not clearly specified:  

It says 
- "_Range / Valid values: 1 - 1000s_", then seconds as unit.
- "_Picture frames must contain motion at least the specified number of frames in a row before they are detected as true motion_" so frames as unit?
